app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.kt

```diff
@@ -308,12 +249,12 @@ class SavingsMakeTransferFragment : BaseFragment(), SavingsMakeTransferMvpView {
     }
 
     override fun showProgress() {
-        layoutMakeTransfer?.visibility = View.GONE
+        binding.llMakeTransfer?.visibility = View.GONE
```

Comment:
In "binding.llMakeTransfer?.visibility" please remove the unnecessary safe-call.